test/test-slowparse.js

```diff
@@ -129,7 +129,7 @@ module.exports = function(Slowparse, window, document, validators) {
     equal(script.childNodes.length, 1, "<script> has one child");
     var textNode = script.childNodes[0];
 
-    equal(textNode.nodeType, textNode.TEXT_NODE,
+    equal(textNode.nodeType, 3,
```

Comment:
always name your magic constants, or people won't have any idea what's going on: at the top of test-slowparse, let's define a var like `var TEXT_NODE = 3;` and then refer to that here. That way the code is immediately obvious, and the value's no longer a magic number.